src/tools/miri/tests/native-lib/scalar_arguments.c

```diff
@@ -28,4 +24,4 @@ short add_int16(short x) {
 
 long add_short_to_long(short x, long y) {
   return x + y;
-}
+}
```

Comment:
Missing newline at end of file

Reply:
Yeah, having the formatter work would've been nice... I'll fix the newlines.